invenio_records_rest/views.py

```diff
@@ -205,8 +206,9 @@ def create_url_rules(endpoint, list_route=None, item_route=None,
     :param record_class: A record API class or importable string used when
         creating new records.
     :param record_serializers: Serializers used for records.
-    :param record_serializers_aliases: A mapping of query arg `format` values
-        to valid mimetypes: dict(alias -> mimetype).
+    :param record_serializers_aliases: A mapping of values of the defined
+        query arg (see `config.REST_MIMETYPE_QUERY_ARG_NAME`) to valid
+        mimetypes for record item serializers: dict(alias -> mimetype).
     :param record_loaders: It contains the list of record deserializers for
         supperted formats.
```

Comment:
typo spotted :) if you want to fix it too ;)

Reply:
Done! 👍